conda_libmamba_solver/solver.py

```diff
@@ -636,6 +636,11 @@ def _package_info_to_package_record(
         else:
             url = pkg.package_url
         url = percent_decode(url)
+        # not all versions of libmambapy support this attribute, note that getattr cannot be used
```

Comment:
Interesting! What's wrong with `getattr`? 🤔

Reply:
`getattr` attempting to access the attribute without catching the AttributeError:
```
(Pdb++) pkg
<libmambapy.bindings.specs.PackageInfo object at 0x10280b6b0>
(Pdb++) getattr(pkg, "boom")
*** AttributeError: 'libmambapy.bindings.specs.PackageInfo' object has no attribute 'boom'
```

I don't know if this is the default in pybind11 or specific to the PackageInfo class. 
I'll update the commend to mention this error.

Reply:
I forgot that `getattr` requires a default value to not raise an AttributeError. `getattr` with a default is fine to use here.

Reply:
Sounds good!